### DISCUSSION TITLE
OpenPDF not working on Gradle

### DISCUSSION THREAD

**Discussion by robertparnpuu** [LibrePDF/OpenPDF#883] [Q&A]
I am using OpenPDF 1.3.30 and I need to use it in a Gradle project with IntelliJ IDEA Community. The dependency seems to resolve all right, but is not recognized within code. I have tried reloading Gradle projects and invalidating caches and restarting but nothing seems to work. Tried with new projects on two different computers as well but still, IDEA does not recognize it in text, nor imports. It seems to be a problem only with OpenPDF, as other dependencies work fine. Could I please get some help?

**Comment by andreasrosdalw** [comment]
Pull request for Gradle support fix is welcome.

**Comment by asturio** [comment]
Any news about this? I would think, that the problem was with gradle, and not OpenPDF, as a jar is a jar. Or maybe a problem with the IDE.

Does your project compiles from the command line?
